code/modules/events/market_crash.dm

```diff
@@ -11,33 +11,46 @@
 	description = "Temporarily increases the prices of vending machines."
 
 /datum/round_event/market_crash
-	var/market_dip = 0
+	var/every_fifth = 1
```

Comment:
Can you document this var? It's not very self-explanatory.

Reply:
Maybe also just rename it to something like "tick counter" and make a define for how many times we want it to tick before doing our thing tbh.